Samples~/SampleGame/Assets/Scripts/Server/GameLiftServer.cs

```diff
@@ -79,7 +79,7 @@ public void Start(int port, string authToken = null, string logFilePath = null)
             computeName = _settings.GetSetting(ServerSettingsKeys.ComputeName).Value;
 #endif
             var serverParams =
-                new ServerParameters(websocketUrl, $"{Application.productName}-{Guid.NewGuid()}", computeName, fleetID,
```

Comment:
Could you elaborate what this change is supposed to do?

Reply:
I don't see how removing the prefix solves the uniqueness issue since we are suffixing it with a unique guid

Reply:
I was receiving warnings when the productID included invalid characters for processId

Reply:
Added a prefix "Amazon-GameLift-SampleGame"